Library/Homebrew/formula_installer.rb

```diff
@@ -1360,15 +1390,18 @@ def previously_fetched_formula
   def fetch_bottle_tab(quiet: false)
     return if @fetch_bottle_tab
 
-    begin
-      formula.fetch_bottle_tab(quiet: quiet)
-      @bottle_tab_runtime_dependencies = formula.bottle_tab_attributes
-                                                .fetch("runtime_dependencies", []).then { |deps| deps || [] }
-                                                .each_with_object({}) { |dep, h| h[dep["full_name"]] = dep }
-                                                .freeze
-    rescue DownloadError, Resource::BottleManifest::Error
-      # do nothing
+    if (download_queue = self.download_queue) &&
+       (bottle = formula.bottle) &&
+       (manifest_resource = bottle.github_packages_manifest_resource)
+      download_queue.enqueue(manifest_resource)
+    else
+      begin
+        formula.fetch_bottle_tab(quiet: quiet)
+      rescue DownloadError, Resource::BottleManifest::Error
+        # do nothing
+      end
     end
```

Comment:
Need to incorporate the retry logic in https://github.com/Homebrew/brew/blob/e8048606d1408678b1e375e8c2c70b797bb5a006/Library/Homebrew/bottle.rb#L123-L132 since we're not calling `fetch_bottle_tab` anymore. Perhaps by having a custom error callback for the queue?

The final `rescue DownloadError, Resource::BottleManifest::Error` here is also important for CI when we enable it there.

Reply:
https://github.com/Homebrew/brew/pull/20288